docs: add info on new api lb routing modes

Crispy1975 opened the pull request:
## What kind of change does this PR introduce?
Docs update for incoming API load balancer routing mode.

https://docs-ou07fwklb-supabase.vercel.app/docs/guides/platform/read-replicas#experimental-routing

Crispy1975 committed:
docs: add info on new api lb routing modes

tylerfontaine committed:
update GUC links to official postgres docs (#29801)

* update GUC links to official postgres docs

* fix copy/paste errors

Crispy1975 commented on apps/docs/content/guides/platform/read-replicas.mdx:
```diff
@@ -102,6 +102,15 @@
 
+#### Experimental Routing
+
+The API load balancer offers an experimental routing mode that build on top of the existing funtionality. It is now possible to use your load balancer endpoint for all Supabase related traffic. The following list summarises the features that are now accessible.
```
Fixed

Crispy1975 commented on apps/docs/content/guides/platform/read-replicas.mdx:
```diff
@@ -102,6 +102,15 @@
+- Auth requests are sent to the Primary database even when using the load balancer endpoint.
+- Routing of Edge Functions, Storage, and Realtime requests to the appropriate backend Supabase service.
+
+The experimental features listed can be enabled by sending the header `sb-lb-routing-mode: alpha-all-services` along with any requests. This gives the option to opt-in and try the new features out, but still have the peace of mind that the original behaviour is there should it be needed. Please let us know if you have any feedback by [contacting us](https://supabase.com/support).
```
Fixed.

charislam committed:
fix(docs): redirects (#29816)

Fix some incorrect or obsolete redirects.

There may be problems for some people whose browsers are caching the old redirects, but we can cache-bust that separately if needed.

avallete committed:
fix(studio): datetime formfield components field update (#29823)

* fix(studio): datetime formfield components field update

gregnr committed:
Blog: database.build live share (#29820)

* blog: database.build live share

* fix: canadian spelling

* feat: update post title/description

* fix: typos

* feat: add open source section to post

* feat: og and thumb images for post

* blog: title/wording tweaks

* blog: reverse proxy wording adjustments

* blog: wording adjustments

* fix: julien blog author

gregnr committed:
Rename Live Share blog post (#29827)

* feat: rename live share blog post

* feat: update casing on post title

darora commented on apps/docs/content/guides/platform/read-replicas.mdx:
```diff
@@ -102,6 +102,15 @@ To call a read-only Postgres function on Read Replicas through the REST API, use
+The API load balancer offers an experimental routing mode that builds on top of the existing funtionality. It is now possible to use your load balancer endpoint for all Supabase related traffic. The following list summarizes the features that are now accessible.
+
+- Auth requests are sent to the Primary database even when using the load balancer endpoint.
+- Routing of Edge Functions, Storage, and Realtime requests go to the appropriate backend Supabase service.
```
While internally this is a routing problem in terms of implementation, from the user's perspective it's really more of a "The LB endpoint now supports usage of additional services (Auth, Storage, Realtime, Edge Functions)."

Can add a note that "Due to the requirements of the Auth service, all Auth requests are handled by the Primary, even when sent over the LB endpoint, similar to how non-Read requests for the Data API (Postgrest) are exclusively handled by the Primary."

Crispy1975 replied:
Updated.

pcnc committed:
feat: support for multiple pg versions (#29473)

* feat: support multiple pg versions

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

jordienr committed:
add info box to let users know they can send an email to support (#29780)

* add info box to let users know they can send an email to support

* make better

* move projects to tooltip

* truncate longer proj names

* fix things properly

* rm always open

* bit more padding

* add mailto link

* Add font-mono to project reference span

* more padding, fix bg hover

pcnc committed:
fix: button text when unpausing project (#29838)

charislam committed:
feat(db, docs): prep to sync troubleshooting entries to github (#29824)

Add JSON schema validation for troubleshooting entries, and add columns for storing GitHub discussion metadata.

Also some minor UI tweaks for troubleshooting display.

fsansalvadore committed:
mongodb blog post (#29795)

* mongodb blog post

* learn how

* add form link

charislam committed:
fix(docs): ref scroll (#29819)

Fix various scrolling bugs with reference pages:

- Take out smooth scrolling as it tends to lead to buggy scrolling position
- Add overscroll-behavior: contain to the nav, otherwise overscrolling on the nav scrolls the main page, which causes the nav to jump around unexpectedly
- Put aria-current calculation in useEffect as otherwise it doesn't trigger properly on first load

staaldraad committed:
chore: remove bruno from humans.txt (#29845)

jan-tennert committed:
docs: Update Kotlin OTP link verification docs (#29839)

Update OTP link verification section

mansueli committed:
feat: improve timeouts for webhooks (#29066)

t6tv8e committed:
Docs // Typo fix // seeding-your-database.mdx (#29822)

Typo fix // Update seeding-your-database.mdx

J0 committed:
fix: update curl snippet for invite user (#29851)

fix: update snippet

kevcodez committed:
chore: remove disk exceeding global header (#29858)

Temporarily remove until we have better usage insights - we still show overages on the usage page, just not the aggressive top level banner

PLyn committed:
Update webhook label to new maximum of 10000ms (#29861)

* Update webhook label to new maximum of 10000ms

* Match formatting of validation label

diamondalltheway committed:
fix: typo in General.tsx (#29487)

MildTomato committed:
chore/ fix-www-blog-cat-build-issue (#29871)

rename cats

kevcodez committed:
feat: new disk size usage section (#29862)

kevcodez committed:
fix: disk size overages (#29875)

jordienr committed:
update collection form (#29791)

* add component for collection form

* use new comp in create collection

* update updatecollection query

* add comp to update collection dialog

* add missing ttype

ivasilov committed:
fix: Fix the snippets loading in local environment (#29750)

Add api routes to make the snippets v2 "work" as it did before.

saltcod committed:
Chore/search user json (#29825)

* Add search to raw json panel

* Cleanup

* Update apps/studio/components/interfaces/Auth/Users/UserPanel.tsx

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

---------

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

awalias committed:
fix: consistency (#29873)

J0 committed:
fix: change to return jsonb (#29887)

delgado3d committed:
fix: typo in upgrade page (#29882)

joshenlim committed:
Add credentials include in all fetchers (#29834)

joshenlim committed:
Update auth hooks to use new endpoint and add perms checking (#29870)

* Update auth hooks to use new endpoint and add perms checking

* Remove unnecessary import

* Update perms checking for edge functions secrets

joshenlim committed:
Revert "Update auth hooks to use new endpoint and add perms checking" (#29893)

Revert "Update auth hooks to use new endpoint and add perms checking (#29870)"

This reverts commit a3e1fb2a0e40a5aedac13be6813f336a56a81770.

joshenlim committed:
Update PH consent banner (#29891)

* Update PH consent banner

* Fix

sayil committed:
Update humans.txt (#29885)

saltcod committed:
Chore/cron nudges (#29797)

* Remove beginning empty state

* Change modal to destructive

* Reorder dropdown menu

* Spacing

jgoux committed:
docs: remove create an organization recommendation in supabase integration (#29898)

ivasilov committed:
fix: Don't send the request twice when saving a policy from the Policy editor (#29899)

Don't send the request twice when saving a policy from the Policy editor.

fsansalvadore committed:
add webinar link to auth page (#29907)

add callout to auth page

joshenlim committed:
Revert "Revert "Update auth hooks to use new endpoint and add perms checking"" (#29894)

Revert "Revert "Update auth hooks to use new endpoint and add perms checking"…"

This reverts commit a2a94d1f7402d07dbfd43246b46b562be140f05c.

ivasilov committed:
chore: Add Ivan to humans.txt (#29902)

Update humans.txt

joshenlim committed:
Update telemetry calls for docs to support PH (#29749)

* Update telemetry calls for docs to support PH

* Update telemetry calls for www to support PH

* Add ts ignore

* Remove use of useRouter for docs

* Add credentials include for www and docs for telemetry calls

* Update TELEMETRY_CONSENT in common for www and docs to reset telemetry opt in preference

* Update common telemetry to use new endpoint and payload, and trigger reset request when opting out of telemetry from ui patterns PrigacySettings

* Fix

* Fix

* Fix

* Fix build issue in docs

* Fix build issue in docs

* I hope this fixes the build issues

* once more...

* Fix

* Add credentials include

* Fix

w3b6x9 committed:
fix: fully qualify table in send email auth hook (#29906)

joshenlim committed:
Add telemetry page leave for www and docs (#29914)

* Add telemetry page leave for www and docs

* Feex

alaister committed:
chore: make dependabot happy 16/10/24 (#29918)
